src/main/java/org/group23/SurveyRepository.java

```diff
@@ -0,0 +1,8 @@
+package org.group23;
+
+import org.springframework.data.repository.CrudRepository;
+
+public interface SurveyRepository extends CrudRepository<Survey, Long> {
+
+    Survey findByid(Long id);
```

Comment:
Same comment - check if findByid vs findById

Reply:
I made them findByid because the other one was giving an error, turns out its because findById is already provided and this line does nothing. Will fix in next PR.